### DISCUSSION TITLE
Implement split workspaces as option without a plugin

### DISCUSSION THREAD

**Discussion by nonetrix** [hyprwm/Hyprland#4000] [General]
I really like Hyprland but personally hate how it handles workspaces and find it confusing, the only reason I use plugin is to get split workspaces and I'd rather not to avoid breakage besides that Hyprland is perfect out of the box. I think this a common enough use case that it should be included OTB in my personal opinion for people that want it. Here is what I use currently: https://github.com/Duckonaut/split-monitor-workspaces

Someone might have already suggested this, but I could not find such a discussion or issue personally. I hope this could be added someday, maybe something like this to enable it
```
general {
  split-workspaces = true
}
```

**Comment by vaxerski** [comment]
my policy isn't to include everything possible in core. 

> I think this a common enough use case that it should be included OTB

lol

**Comment by nonetrix** [comment]
Sure for things that like 0.0000000001% of users would ever use, but something like this maybe should be included. But that's just my opinion, you don't have to listen to me

**Reply by vaxerski** [reply]
for now my slightly opinionated development strategies have propelled this project to be the number 1 most trending wayland compositor for over a year straight and counting. I'll stick to my practices.

Plugins are there for a reason, and are getting easier and easier to use. hyprload exists, too.
